From 0773caae1b4eae198f8e41fb8a1d46c24c99543e Mon Sep 17 00:00:00 2001
From: Tim Waugh <twaugh@redhat.com>
Date: Thu, 8 Dec 2011 17:32:09 +0100
Subject: [PATCH] '@' is an 'ampersat' not an 'ampersand'; let's call it 'at
 symbol' (cherry picked from commit
 7e115808a990199fe00de4cc2ef48fe96de3d3d0)

---
 man/systemd.socket.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/man/systemd.socket.xml b/man/systemd.socket.xml
index 2f31242..f883543 100644
--- a/man/systemd.socket.xml
+++ b/man/systemd.socket.xml
@@ -156,7 +156,7 @@
                                 family.</para>
 
                                 <para>If the address starts with an
-                                ampersand (@) it is read as abstract
+                                at symbol (@) it is read as abstract
                                 namespace socket in the AF_UNIX
                                 family. The @ is replaced with a NUL
                                 character before binding. For details
